reports/final_mds/finding-fossils-final-mds.qmd

```diff
@@ -0,0 +1,806 @@
+render_df = render_df.melt(id_vars=["Model"], var_name="Metric", value_name="Score")
+render_df["Metric"] = (
+    render_df.Metric.str.replace("Token ", "")
+    .str.replace(" Recall", " (%)", regex=True)
+    .str.replace("Recall", "Recall (%)")
+)
+render_df["Model"] = (
+    render_df.Model.str.replace("Finetuned", "")
+    .str.replace("Spacy", "SpaCy")
+    .str.replace("Finetune", "")
+    .str.replace("V\d", "", regex=True)
+    .str.replace("Transformer", "Transf.")
+    .str.replace("Multilanguage", "Multi-lang.")
+    .str.replace("Ner", "NER")
+    .str.replace("bert", "BERT")
+    .str.replace("Bert", "BERT")
+)
+render_df["Score"] = render_df.Score * 100
+render_df = (
+    render_df.reset_index(drop=True)
+    .round(1)
+    .pivot(index="Model", columns="Metric", values="Score")
+    .sort_values(["Recall (%)"], ascending=False)
+)
+
+render_df = render_df[
+    [
+        "Recall (%)",
+        "Age (%)",
+        "Site (%)",
+        "Taxa (%)",
+        "Region (%)",
+        "Geog (%)",
+        "Email (%)",
+        "Alti (%)",
+    ]
+]
+
+Markdown(render_df.to_markdown())
+```
+
+An important observation to make here is that the top models had a lower precision score for the SITE names and REGION names. The models got confused when deciding whether an entity should be classified as a SITE or a REGION. This was partially due to quality of labeling entities as well as the fact that both these types correspond to the name of a place or a wider area. See @fig-confusion_matrix for a confusion matrix generated using the test set assets highlights the issue.
+
+![Confusion Matrix for RoBERTa Model](../../results/ner/roberta-finetuned-v3/roberta-finetuned-v3_test_confusion_matrix.png){#fig-confusion_matrix}
+
+## Data Review Tool
+
+The final data review tool that was created is a multi-page Plotly Dash [@dash] application. The tool can be replicated by launching Docker containers, enabling anyone within the Neotoma community to easily utilize the tool for reviewing outputs from the pipeline. 
+
+![Data Review Tool](assets/data_review.png){#fig-review_tool_snap}
+
+This web application enables users to review the extracted entities from articles, make changes, add additional missed entities and remove incorrectly extracted entities. The entire review process does not require any coding knowledge, and reviewers can navigate through the review workflow in three clicks or fewer from the Article Review page @fig-review_tool_snap.
+
+The output of this data review tool is a parquet file that stores the originally extracted entities as well as the corrected entities. The Neotoma organization can utilize these entities to add new paleoecology articles to the database. In addition, the reviewed entities can be fed back to model using our model retraining pipeline, contributing to the continuous improvement of the quality of extraction of entities as outlined in @tbl-review_results.
```

Comment:
edit: `The output of this data review tool is a parquet file that stores the corrected entities`